backend/tests/integration/README.md

```diff
@@ -0,0 +1,41 @@
+# Welcome to the backend integration tests
+
+## How to run
+
+```sh
+make -C backend test-integration GOCOVERDIR=/tmp
+```
+
+by default the following tags will be used:
+
+```
+mendersoftware/iot-manager:test
+```
+
+## Test separation
+
+ As of the moment of speaking we ported the compat test from the integration to the backend integration tests.
+This was not that straightforward as it seemed, mainly because the pending devices presence disturbed
+other tests. KJ suggested to run it in a separate pytest session. This also seemed straightforward.
+ If you want to run your test in a separate pytest session, you need to add `+separate` interfix before `.py`.
+See the below example for ``:
+
+```sh
+ls -1 backend/tests/integration/tests/
+test_access.py
+..
+test_compat+separate.py
+..
+test_workflows.py
+```
+
+Further more if you need to have the additional docker-compose files included in your separated run
+you have to define `COMPOSE_FILES_$yourtest_name`, see the example in the `backend/tests/integration/run`,
+for `test_compat+separate.py` the variable is:
+
+```sh
+COMPOSE_FILES_test_compat
+```
+
+where the `test_compat` corresponds to thetest file name without `.py` and `+separate`.
+
```

Comment:
This is a lot of instructions for adding a test, and it is also adding complexity to the entire structure.
How about doing something like [this](https://docs.pytest.org/en/7.1.x/example/markers.html#custom-marker-and-command-line-option-to-control-test-runs)?
Or, add a pytest argument that select only that test and skip all others and use that in the docker compose override?

Reply:
I will do anything to stay away from the pytest mechanisms as they are unportable and really, really obscure and make things _extremely_ confusing. so unless you block it, I will fight for this. all you do is you add "+separate" to the filename, how can this be complicated?

Reply:
so, adding a pytest argument defies the purpose here: with the present patch we have a general mechanism to run any test separately.  all you need to do is add as many test_*+separate.py files and each one of them and out of the box will run completely isolated.